sdk/core/core-http/lib/util/utils.ts

```diff
@@ -89,7 +94,7 @@ export function isValidUuid(uuid: string): boolean {
  *
  * @return {any[]} An array of values of the given object.
  */
-export function objectValues(obj: { [key: string]: any; }): any[] {
+export function objectValues(obj: { [key: string]: any }): any[] {
```

Comment:
This helper can be replaced by using standard Object.values

Reply:
Looks like noone's using this, removing it